### ISSUE TITLE
[World Quest] Val'sharah ID 41955

### ISSUE THREAD

**Issue by Redbaronn** [Evolution-WoW/Bug-Report#1033]
The world quest "Bloodline of Stone" ID 41955 is bugged in a way that you cannot progress further due to "saved" progress from yesterday. So I did this quest yesterday (which worked fine yesterday and finished it) and today is a new reset and I went in to finish again but the problem is that I had some saved progress as in 1/1 Magula slain but 0/5 Harpy nests burned and 0/10 Harpy eggs broken. I did the Harpy nests with no problem but now i faced another problem where the eggs are already broken even though i haven't interacted them to break them in the first place and by thus you cannot progress with the quest and finish the quest.

https://cdn.discordapp.com/attachments/929136522777727056/971046765635715172/unknown.png

https://cdn.discordapp.com/attachments/929136522777727056/971044403835715584/unknown.png - Here you can see the eggs are broken but the item is shining like its ready to be interacted it (The progress is 3/10 eggs broken here)

https://cdn.discordapp.com/attachments/929136522777727056/971044444092633128/unknown.png - And here after they disappeared like they've been broken (also i never interacted with them just went near them to make the screenshot more closer for the shine to be more visible and they just disappared). And also the progress is the same which is 3/10.

https://cdn.discordapp.com/attachments/929136522777727056/971044875493605456/unknown.png - Same scenario here, 5 eggs prebroken and the progress doesn't count.

https://cdn.discordapp.com/attachments/929136522777727056/971044909761052722/unknown.png - And after that they just disappear even without interacting with them.

So to sum things up: the world quest wasn't affected by the daily reset and the progress was saved by thus its not repeatable on a daily basis.
